src/ray/common/scheduling/scheduling_ids.h

```diff
@@ -0,0 +1,50 @@
+#ifndef RAY_COMMON_SCHEDULING_SCHEDULING_IDS_H
+#define RAY_COMMON_SCHEDULING_SCHEDULING_IDS_H
+
+#include <iostream>
+#include <map>
+#include <unordered_map>
+#include <string>
+#include <iterator>
+#include <stdlib.h>
+#include <chrono>
+#include <vector>
+using namespace std;
```

Comment:
We shouldn't open the namespace `std` like this (it would open it for every file that includes this file). Instead, use `std::`.

Reply:
fixed